includes/Core/Tags/GTag.php

```diff
@@ -188,4 +186,18 @@ function( $arg ) {
 
 		return sprintf( 'gtag(%s);', implode( ',', $gtag_args ) );
 	}
+
+	/**
+	 * Returns the gtag source URL.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string The gtag source URL.
+	 */
+	public function get_gtag_src() {
```

Comment:
As a public method, please add test coverage.

For example, this looks like it would error if called with no `tags` added.

Reply:
This got missed, my bad. Added now, thanks!